packages/fx-core/src/plugins/resource/bot/v2/mapping.ts

```diff
@@ -51,23 +54,23 @@ export function getRuntime(lang: ProgrammingLanguage): string {
   if (runtime) {
     return runtime;
   }
-  throw new Error("invalid bot input");
+  return defaultLang;
 }
 
-export function getServiceType(hostType: string): ServiceType {
-  const serviceType = serviceMap.get(hostType);
+export function getServiceType(hostType?: string): ServiceType {
+  const serviceType = serviceMap.get(hostType ?? "");
   if (serviceType) {
     return serviceType;
   }
-  throw new Error("invalid bot input");
+  return defaultServiceType;
 }
 
-export function getLanguage(lang: string): ProgrammingLanguage {
-  const language = langMap.get(lang.toLowerCase());
+export function getLanguage(lang?: string): ProgrammingLanguage {
+  const language = langMap.get(lang?.toLowerCase() ?? "");
   if (language) {
     return language;
   }
-  throw new Error("invalid bot input");
```

Comment:
Some unit test does not mock language, host type, etc. Throwing error breaks those ut.
Return default value to work around. Let's fix those ut later.

Reply:
Sure, let's use this workaround first to avoid breaking ut, and then add todo later to fix ut and convert the code back to throw exception. @yukun-dong can you help fix the ut later?

Reply:
Yes, of course.